arviz/sel_utils.py

```diff
@@ -0,0 +1,224 @@
+"""Utilities for selecting and iterating on xarray objects."""
+from itertools import product, tee
+
+import numpy as np
+import xarray as xr
+
+
+def selection_to_string(selection):
+    """Convert dictionary of coordinates to a string for labels.
+
+    Parameters
+    ----------
+    selection : dict[Any] -> Any
+
+    Returns
+    -------
+    str
+        key1: value1, key2: value2, ...
+    """
+    return ", ".join(["{}".format(v) for _, v in selection.items()])
+
+
+def make_label(var_name, selection, position="below"):
+    """Consistent labelling for plots.
+
+    Parameters
+    ----------
+    var_name : str
+       Name of the variable
+
+    selection : dict[Any] -> Any
+        Coordinates of the variable
+    position : str
+        Whether to position the coordinates' label "below" (default) or "beside"
+        the name of the variable
+
+    Returns
+    -------
+    label
+        A text representation of the label
+    """
+    if selection:
+        sel = selection_to_string(selection)
+        if position == "below":
+            base = "{}\n{}"
```

Comment:
This looks bad with numbers, but good with strings

Reply:
yep, any ideas on how to handle this are very welcome, having these 2 different defaults (one for plots and another for summary) also makes it harder (at least for me) to think about sensible ways to change the labeling function